ts/modules/bullet.src.ts

```diff
@@ -304,21 +304,33 @@ seriesType<Highcharts.BulletSeries>('bullet', 'column'
         getExtremes: function (
             this: Highcharts.BulletSeries,
             yData?: Array<number>
-        ): void {
+        ): Highcharts.DataExtremesObject {
             var series = this,
                 targetData = series.targetData,
                 yMax,
                 yMin;
 
-            columnProto.getExtremes.call(this, yData);
+            const dataExtremes = columnProto.getExtremes.call(this, yData);
 
             if (targetData && targetData.length) {
-                yMax = series.dataMax;
-                yMin = series.dataMin;
-                columnProto.getExtremes.call(this, targetData);
-                series.dataMax = Math.max(series.dataMax, yMax);
-                series.dataMin = Math.min(series.dataMin, yMin);
+                const targetExtremes = columnProto.getExtremes.call(
+                    this,
+                    targetData
+                );
+                if (typeof targetExtremes.dataMin === 'number') {
```

Comment:
Wouldn't be better to use `isNumber()`? We get for free checking for `NaN` and if value is finite.

Reply:
I agree, but I wonder why it is optional in the first place. Shouldn't DataExtremesObject always provide these numbers?

Reply:
They can be undefined, see comment below.